docs/_posts/0000-05-01-oo-inheritance.md

```diff
@@ -178,18 +178,21 @@ classDiagram
 + It tends to come with a rigid set of rules
 + A more common way to model relationships between objects is via _associations_
 + An _association_ is just when one class references another
++ A special type of _association_ is `composition`
+
+Note: similarly aggregation is also association where many objects are aggregated (stored in a collection) by another object.   
 
 --
 
 ### Associations between our card classes
 
 ```java
 class PlayingCard extends Card {  //inheritance
-    private Suit suit;  //association
+    private Suit suit;  //association (composition)
```

Comment:
I'm a bit wary of overloading them with terms - not sure we really care about the different types of association for now, seems a bit too detailed for our purposes.